src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -325,7 +325,7 @@ private bool LoadFile(string path)
                 XmlSerializer serializer = new XmlSerializer(typeof(AutoVisualizer));
                 if (!File.Exists(path))
                 {
-                    _process.WriteOutput(String.Format(CultureInfo.CurrentCulture, ResourceStrings.FileNotFound, path));
```

Comment:
does Logger support formatting using CurrentCulture? I think the logger was using InvariantCulture everywhere I looked

Reply:
Good catch. I will fix this is HostLogChannel.